vendor/github.com/grafana/dskit/spanlogger/spanlogger.go

```diff
@@ -101,7 +101,7 @@ func withContext(ctx context.Context, logger log.Logger, resolver TenantResolver
 	// even though the code-base generally uses `userID` to refer to the same thing.
 	userID, err := resolver.TenantID(ctx)
 	if err == nil && userID != "" {
-		logger = log.With(logger, "org_id", userID)
+		logger = log.With(logger, "user", userID)
```

Comment:
Given this change, do you mind adding this PR number to the following CHANGELOG entry, please?

```
* [CHANGE] All: Logs that used the`org_id` label now use `user` label. #1634
```

Reply:
Good call, will fix.